integration/mediation-tests/tests-mediator-2/src/test/resources/artifacts/ESB/mediatorconfig/datamapper/one_to_one/xml_to_xml_not_xslt_compatible/xsltStyleSheet.xml

```diff
@@ -0,0 +1 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?><xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" xmlns:notXSLTCompatible="true" xmlns:own="http://ownfunction.org" xmlns:xs="http://www.w3.org/2001/XMLSchema" version="2.0"><xsl:function name="own:setPrecision"><xsl:param name="resultString"/><xsl:param name="noOfDigits"/><xsl:if test="$noOfDigits=0"><xsl:value-of select="$resultString"/></xsl:if><xsl:if test="$noOfDigits!=0"><xsl:value-of select="own:setPrecision(concat($resultString,'0'),$noOfDigits - 1)"/></xsl:if></xsl:function></xsl:stylesheet>
```

Comment:
Shall we format this? Instead of having everything in a single line?

Reply:
This is a generated resource by developer studio. So do i need to change it?